src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -25,6 +28,7 @@ type Props = {
 };
 
 const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Props) => {
+    const showLimit = versions.length >= MAX_VERSIONS;
```

Comment:
Should this take into account the `versionLimit` prop? Is this passed in on a per enterprise basis?

Reply:
Talked about this offline with @ConradJChan